style.css

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital@0;1&display=swap');
+
+html {
+  box-sizing: border-box;
+  font-size: 10px;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  font-size: 1.5rem;
+}
+
+.headline-wrapper {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  border-bottom-left-radius: 110px;
+  background-image: url(file:///C:/Users/VS47~1/AppData/Local/Temp/header_section_background.png);
```

Comment:
- [ ] This `image` is broken because its path points to a folder on your personal computer not the repository, I kindly suggest you to add the `image` to your repository and then fix the path 👍🏻